docs/content/preview/explore/connection-manager/connection-mgr-ysql.md

```diff
@@ -0,0 +1,87 @@
+
+To start a YugabtyeDB cluster with the YSQL Connection Manager, set the [yb-tserver](../../../reference/configuration/yb-tserver/) flag `enable_ysql_conn_mgr` flag to true.
+After `enable_ysql_conn_mgr` is set, yb-tserver will start the YSQL Connection Manager process along with the PostgreSQL process. You should see one YSQL Connection Manager process per yb-tserver.
+
+To start a single node cluster with YSQL Connection Manager, use the following [yugabyted](../../../reference/configuration/yugabyted/) command:
+
+```sh
+./bin/yugabyted start --tserver_flags "enable_ysql_conn_mgr=true,allowed_preview_flags_csv=enable_ysql_conn_mgr" --ui false
+```
+
+`enable_ysql_conn_mgr` is a preview flag only, so to use it, this flag needs to be added in the `allowed_preview_flags_csv list`, that is, `allowed_preview_flags_csv=enable_ysql_conn_mgr`.
+
+{{< note >}}
+
+To create a large number of client connections, ensure that "SHMMNI" (the maximum number of concurrent shared memory segments an OS allows) as well as [ulimit](../../../deploy/manual-deployment/system-config/#ulimits) is set correctly as follows:
+
+1. Open the file `/etc/sysctl.conf`, and add `kernel.shmmni` = `32768` (support for 30000 clients) at the end of the file.
+1. To refresh the settings, use `sudo sysctl -p`.
+{{< /note >}}
+
+### YSQL Connection Manager ports and flags
+
+By default, when YSQL Connection Manager is enabled, it uses the port 5433, and the backend database is assigned a random free port.
+To explicitly set a port for YSQL, you should specify ports for the flags `ysql_conn_mgr_port` and [ysql_port](../../../reference/configuration/yugabyted/#advanced-flags).
+
+The following table describes yb-tserver flags related to the YSQL Connection Manager:
+
+| flag | Description | Default value |
+|:---- | :---------- | :------------ |
+| enable_ysql_conn_mgr | Enables YSQL Connection Manager for the cluster. yb-tserver starts a YSQL Connection Manager process as a child process. | false |
+| ysql_conn_mgr_idle_time | Specifies the maximum idle (secs) time allowed for database connections created by the YSQL Connection Manager. If a database connection remains idle without serving a client connection for a duration equal to, or exceeding the value provided, it will be automatically closed by the YSQL Connection Manager. | 60 |
+| ysql_conn_mgr_max_client_connections | Maximum number of concurrent database connections YSQL Connection Manager can create per pool. | 10000 |
+| ysql_conn_mgr_min_conns_per_db | Minimum number of physical connections that will be present in the pool. This limit is not considered while closing a broken physical connection. | 0 |
+| ysql_conn_mgr_num_workers | Number of worker threads used by YSQL Connection Manager. If set to 0, the number of worker threads will be half of the number of CPU cores. | 0 |
+| ysql_conn_mgr_password | Password to be used by YSQL Connection Manager while creating database connections. | 0 |
```

Comment:
@aishwarya24  can you once update the default values

Reply:
done